src/csstree/CssTree/__init__.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function
+
+import logging
+import CssElement
+
+
+
+
+class CssTree:
+    def __init__(self, min_zoom=1, max_zoom=19):
+        self.subtrees_by_zoom = {}
+        for i in range(min_zoom, max_zoom + 1):
```

Comment:
`self.subtrees_by_zoom = {i: CssSubtree() for i in xrange(min_zoom, max_zoom + 1)}`